### ISSUE TITLE
Prevent "Cannot find a reference" warnings

### ISSUE THREAD

**Issue by extsoft** [CiscoTestAutomation/pyats#8]
There is a pretty simple code:
```python
from ats.topology import loader
cluster = loader.load('./testbeds/test.yaml')
```
And it works well. 

From the other side, this code generates a lot of warnings from IDEs or static analysis. For instance, `pylint` will generate the following error: `E1101: Module 'ats.topology.loader' has no 'load' member (no-member)`.

I believe this happens as pyATS is distributed via binary format and either `pylint` or `PyCharm` are not able to understand the runtime members of a module. As the result, either code completion nor static analysis are working properly.

Looks like a solution is the stub files - https://www.python.org/dev/peps/pep-0484/#stub-files. Maybe there are any other options we can discuss. 

Anyway, I'm ready to contribute in this activity some development efforts.

**Comment by siming85** [comment]
most likely because 

ats.topology.loader module is replaced as an instance of the Loader class in sys.modules. That tends to cause pylint to panic.

**Comment by extsoft** [comment]
Looks like. Anyway, the stub files have to solve this problem without changing a code. 
@siming85 what do you think about? Are there other options?

**Comment by siming85** [comment]
isn't stub files only part of Python3.5?
we could change the implementation slightly so it doesn't yell at you like that - it'll be an easier fix.

atm we still support python 3.4, so stubs won't fix that.

**Comment by siming85** [comment]
this should be fixed in the next release.

**Comment by siming85** [comment]
fixed in pyATS release v5.0.0, available as of Oct 9
